### PR TITLE
Update generators.js

### PR DESCRIPTION
**Pull request by ghost**
Cloudy Cloud function has been added :)

### CODE REVIEW

**Comment by JonathanHHenson** [comment]
The cloud is too small, and you shouldn't change the background. Your cloud must also return a bounding box so that the name is displayed on top.

**Comment by meiamsome** [comment]
Just an FYI you don't need to push/pop at the start/end and you don't need to call noLoop

**Comment by ghost** [comment]
Thanks you guys for the heads up, I did as requested :)

File: generators.js [hunk 1]
```diff
@@ -1244,6 +1244,21 @@ function noCloud() {
...
+	//I'm really sorry about this one xD
+	translate(width/2,height/2);
+	noStroke();
+	for(let i = 0; i < 30; i++)
+		ellipse(random(-130,130),random(-15,25),random(50,200));
+  
+  return [-130,-50,130*2,50*2];
+}
+register(CloudyCloud, "Radnomly Generated Cloud", "this.Zohir")
+=======
```
Comment (meiamsome) [inline generators.js]: Looks like this is left over from the merge resolution. Is that something you can fix easily? If not I'll fix it whilst merging!

Comment (ghost) [reply generators.js]: I don't understand, the return statement is mine

Comment (meiamsome) [reply generators.js]: Sorry, this wasn't very clear - I was commenting baout the line 1261: ======= - it's a remainder from the merge resolution you did. Don't worry, I'll fix it and merge :)

**Pull request merged**